svc-systems/rpc/chassis.go

```diff
@@ -59,6 +59,24 @@ type ChassisRPC struct {
 	CreateHandler        *chassis.Create
 }
 
+func (cha *ChassisRPC) UpdateChassisResource(_ context.Context, req *chassisproto.UpdateChassisResourceRequest) (*chassisproto.GetChassisResponse, error) {
```

Comment:
Remove context from func signature (looks like its not used)

Reply:
This method's signature is defined in ChassisServer interface generated by protoc gen go chassis.pb.go, and as an argument it takes context.